### DISCUSSION TITLE
functionCallVolU and functionCallVolD calls do not change volume

### DISCUSSION THREAD

**Discussion by maxbobse** [MiczFlor/RPi-Jukebox-RFID#1971] [Q&A]
I have set up my buttons to lower and raise the volume and configured the correct pins in the gpio_settings.ini file. But when actually pressing the buttons, nothing changes.

I have other buttons configured to Play/Pause and Prev/Next and those work fine. When change the call there to change the volume instead, again nothing happens. So the physical setup itself seems to be fine.

What are the best options here to troubleshoot? :)

Thanks!

**Comment by AlvinSchiller** [comment]
Hi, this could be equal to https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2023

Can you try and change your audio_output configuration of mpd.conf?

Set the configuration of "mixer_type" from "hardware" to "software".
